lib/massager_parameters_example.ex

```diff
@@ -0,0 +1,55 @@
+defmodule MassagerParametersExample do
```

Comment:
This is my original thought of how I would make the template work. The user would take this template and make a custom copy for their specific import. But this is a very cumbersome way to do things.